Hazel/src/Hazel.h

```diff
@@ -9,7 +9,6 @@
 #include "Hazel/Core/Timestep.h"
 
 #include "Hazel/Input.h"
-#include "Hazel/KeyCodes.h"
 #include "Hazel/MouseButtonCodes.h"
```

Comment:
Here you either: omit the inclusion of MouseButtonCodes or don't remove the KeyCodes include. I would opt for not relying on other files including it, and go with the latter solution